output/cloud/output_test.go

```diff
@@ -148,21 +149,8 @@ func runCloudOutputTestCase(t *testing.T, minSamples int) {
 	t.Logf("Random source seeded with %d\n", seed)
 
 	tb := httpmultibin.NewHTTPMultiBin(t)
-	tb.Mux.HandleFunc("/v1/tests", http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Considering this test seems to very precisely test just `http_req_li_all`, which is something very specific to the v1 cloud output, do you actually need to make any changes to it? It won't be useful at all for v2, so can't it just directly be moved to v1 as it is?

Reply:
The idea is to move in v1 and v2 only the logic for aggregate/flush (the logic not common) and to keep centralized the start/stop operations. To do it all the tests asserting specific logic will be moved to the appropriate package. So `v1`/`v2` will be not able to resolve the reference ID, they need it injected.

For this reason, we need to not invoke the test creation from all the tests that we plan to move to the specific versioned package.